pos_product_packaging_multi_barcode/static/src/js/db.js

```diff
@@ -0,0 +1,19 @@
+odoo.define("pos_product_packaging_multi_barcode.db", function (require) {
+    "use strict";
+
+    var PosDB = require("point_of_sale.DB");
+
+    PosDB.include({
+        add_packagings: function (packagings) {
+            var res = this._super(packagings);
+            var self = this;
+            packagings.forEach(function (packaging) {
```

Comment:
In core pos module there's a check on if the packaging is linked to a product that is already loaded....
Instead of iterating on all `packagings` wouldn't better to iterate on `this.product_packaging_by_barcode` which is already filtered?

Reply:
Thank @simahawk , I updated